### DISCUSSION TITLE
Deprecation Warnings

### DISCUSSION THREAD

**Discussion by eladavron** [fengsp/flask-session#249] [Q&A] [answered]
On Python `3.11.9`, using `Flask-Session==0.8.0`, I started getting these warnings:

```
.venv/lib/python3.11/site-packages/flask_session/filesystem/filesystem.py:57: DeprecationWarning: 'SESSION_FILE_DIR' is deprecated and will be removed in a future release. Instead pass FileSystemCache(directory, threshold, mode) instance as SESSION_CACHELIB.

.venv/lib/python3.11/site-packages/flask_session/filesystem/filesystem.py:75: DeprecationWarning: FileSystemSessionInterface is deprecated and will be removed in a future release. Instead use the CacheLib backend directly.
```

**Comment by ThiefMaster** [comment]
What's your question? The deprecation warning actually tells you what to do...

> FileSystemSessionInterface is deprecated and will be removed in a future release. **Instead use the CacheLib backend directly.**

**Reply by eladavron** [reply]
It was posted more as a heads up, not a question.
I would fix it myself and submit a pull request (and I still might) if I have the time, but given that I don't at the moment, I posted it here in case anyone else wants to tackle it itmt.

**Reply by ThiefMaster** [reply, marked as answer]
But there is nothing to fix in this library. It deprecated something you are using, so you need to adapt your own code (or more likely config) to avoid using the deprecated code...

**Reply by eladavron** [reply]
Ohhhh I see what you mean now - you are correct.
I'm sorry for the misunderstanding.
Thanks for the quick response!
